tests/integration_tests/dbt_projects/project1_expected/models/example/model_with_custom_configs.sql

```diff
@@ -3,14 +3,14 @@
     meta={'existing_meta_config': 'existing_meta_config', 'custom_config': 'custom_config', 'custom_config_int': 2, 'custom_config_list': ['a', 'b', 'c'], 'custom_config_dict': {'a': 1, 'b': 2, 'c': 3}}
 ) }}
 
-{{ config.get('custom_config') }}
+{{ config.meta_get('custom_config') }}
 
-{{ config.get('custom_config', 'default_value') }}
+{{ config.meta_get('custom_config', 'default_value') }}
 
 {{ config.get('materialized') }}
 
 {{ config.get('materialized', 'default_value') }}
 
-{{ config.get('custom_config_dict').a }}
+{{ config.meta_get('custom_config_dict').a }}
 
-{{ config.get('custom_config_dict').get('a', 'default_value') }}
```

Comment:
Tested this out with a few more edge cases around specifying the 'default' kwarg explicitly:

expected inputs: 
```
{{ config.get('custom_config', default='default_value') }}

{{ config.get('custom_config', default=var.get('my_var')) }}

{{ config.get('custom_config', default=dest_columns | map(attribute=\"quoted\") | list) }}
```

and outputs:
```
{{ config.meta_get('custom_config', default='default_value') }}

{{ config.meta_get('custom_config', default=var.get('my_var')) }}

{{ config.meta_get('custom_config', default=dest_columns | map(attribute=\"quoted\") | list) }}
```

all worked as expected!

would suggest adding these cases as tests though